include/af/opencl.h

```diff
@@ -304,7 +302,7 @@ static inline deviceType getDeviceType()
 
 #if AF_API_VERSION >= 33
 /**
-   Get the type of the current device
+   Get the type of the current platform
```

Comment:
Good catch. It returns an enumeration which identifies the vendor (for lack of a better word) of the OpenCL platform. I've corrected this in my next push.